zipline/data/data_portal.py

```diff
@@ -1284,8 +1284,7 @@ def get_splits(self, sids, dt):
             "SELECT sid, ratio FROM SPLITS WHERE effective_date = ?",
             (seconds,)).fetchall()
 
-        sids_set = set(sids)
-        splits = [split for split in splits if split[0] in sids_set]
+        splits = [split for split in splits if split[0] in sids]
```

Comment:
the `sids` container can be on the order of 500+ sids, so that's why I used a set.  Think we should go back to that?